bazel/repository_locations.bzl

```diff
@@ -227,38 +226,38 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         # This sha on grpc:v1.25.x branch is specifically chosen to fix gRPC STS call credential options.
         version = "d8f4928fa779f6005a7fe55a176bdb373b0f910f",
         sha256 = "bbc8f020f4e85ec029b047fab939b8c81f3d67254b5c724e1003a2bc49ddd123",
+        release_date = "2020-02-11",
         strip_prefix = "grpc-{version}",
         urls = ["https://github.com/grpc/grpc/archive/{version}.tar.gz"],
         use_category = ["dataplane_core", "controlplane"],
-        release_date = "2020-02-11",
         cpe = "cpe:2.3:a:grpc:grpc:*",
     ),
     com_github_luajit_luajit = dict(
         project_name = "LuaJIT",
         project_desc = "Just-In-Time compiler for Lua",
         project_url = "https://luajit.org",
-        # The last release version, 2.1.0-beta3 has a number of CVEs filed
-        # against it. These may not impact correct non-malicious Lua code, but for prudence we bump.
         version = "1d8b747c161db457e032a023ebbff511f5de5ec2",
         sha256 = "20a159c38a98ecdb6368e8d655343b6036622a29a1621da9dc303f7ed9bf37f3",
+        release_date = "2020-10-12",
         strip_prefix = "LuaJIT-{version}",
         urls = ["https://github.com/LuaJIT/LuaJIT/archive/{version}.tar.gz"],
-        release_date = "2020-10-12",
         use_category = ["dataplane_ext"],
         extensions = ["envoy.filters.http.lua"],
         cpe = "cpe:2.3:a:luajit:luajit:*",
+        # The last release version, 2.1.0-beta3 has a number of CVEs filed
```

Comment:
Not sure I find the new comment location (at the bottom) super intuitive, but I don't have a strong opinion.

Reply:
I've moved some the of the comments around to be consistent and now have most of the comments under the `project_url` location. I think this is consistent now